index.js

```diff
@@ -21,6 +23,7 @@ function accesorString(value) {
 
 module.exports = function() {};
 module.exports.pitch = function(remainingRequest) {
+	remainingRequest = '.' + path.sep + path.basename(remainingRequest);
```

Comment:
It might be worth assigning this to a new variable to avoid reassigning to the arg.